src/js/config.js

```diff
@@ -23,6 +23,11 @@ export const MESSAGES = {
 	MSG_5100: '<p>The Republishing Service is currently undergoing maintenance. Please try again later.</p>'
 };
 
+export const RICH_CONTENT_MESSAGES = {
```

Comment:
Because you have created a `messages` files wouldn't all this constants better fit in that file?

Reply:
I decided to leave them where they were because messages.js feels more readable for me as it is. It's a personal preference but happy to move if that is not a shared opinion.